iati/core/tests/test_data.py

```diff
@@ -184,10 +184,13 @@ def test_instantiate_dataset_from_string(self):
         assert isinstance(dataset, iati.core.data.Dataset)
         assert dataset.xml_str == xml
 
-    @pytest.mark.parametrize("encoding", ["UTF-8", "utf-8", "UTF-16", "utf-16"])
+    @pytest.mark.parametrize("encoding", ["UTF-8", "UTF-16", "ASCII",
```

Comment:
Suggest also including each lower-case and with-underscore versions of UTF8 and 16 to show that it isn't case-sensitive